DQM/TrackingMonitorSource/plugins/ttbarEventSelector.cc

```diff
@@ -0,0 +1,405 @@
+      if (beamSpot.isValid()) {
+        double trkd0 = -(tk->dxy(beamSpot->position()));
+        if (std::fabs(trkd0) >= maxD0_)
+          continue;
+        double trkdz = tk->dz(beamSpot->position());
+        if (std::fabs(trkdz) >= maxDz_)
+          continue;
+      } else {
+        edm::LogError("MuonSelector") << "Error >> Failed to get BeamSpot for label: " << bsTag_;
+      }
+
+      const reco::HitPattern& hitp = gtk->hitPattern();
+      if (hitp.numberOfValidPixelHits() < minPixelHits_)
+        continue;
+      if (hitp.numberOfValidStripHits() < minStripHits_)
+        continue;
+
+      // Hits/section in the muon chamber
+      if (mu.numberOfChambers() < minChambers_)
+        continue;
+      if (mu.numberOfMatches() < minMatches_)
+        continue;
+      if (mu.numberOfMatchedStations() < minMatchedStations_)
+        continue;
+      if (!muon::isGoodMuon(mu, muon::GlobalMuonPromptTight))
+        continue;
+
+      // PF Isolation
+      const reco::MuonPFIsolation& pfIso04 = mu.pfIsolationR04();
+      double absiso = pfIso04.sumChargedHadronPt +
+                      std::max(0.0, pfIso04.sumNeutralHadronEt + pfIso04.sumPhotonEt - 0.5 * pfIso04.sumPUPt);
+      if (absiso / mu.pt() > maxIsoMu_)
+        continue;
+
+      TLorentzVector lv_mu;
+      lv_mu.SetPtEtaPhiE(mu.pt(), mu.eta(), mu.phi(), mu.energy());
+      list_mu.push_back(lv_mu);
+      lm = list_mu.size();
+      chrgeList_mu.push_back(mu.charge());
+    }
+    lm = list_mu.size();
+  } else {
+    edm::LogError("MuonSelector") << "Error >> Failed to get MuonCollection for label: " << muonTag_;
+    return false;
+  }
+
+  // for Track jet collections
+  edm::Handle<reco::PFJetCollection> jetColl;
+  iEvent.getByToken(jetsToken_, jetColl);
+  std::vector<TLorentzVector> list_jets;
+
+  if (jetColl.isValid()) {
+    for (const auto& jets : *jetColl) {
+      if (jets.pt() < minPt_Jets_)
+        continue;
+      if (std::fabs(jets.eta()) > maxEta_Jets_)
+        continue;
+      TLorentzVector lv_jets;  // lv_bJets;
+      lv_jets.SetPtEtaPhiE(jets.pt(), jets.eta(), jets.phi(), jets.energy());
+      list_jets.push_back(lv_jets);
+    }
+    lj = list_jets.size();
+  }
+
+  edm::Handle<reco::JetTagCollection> bTagHandle;
+  iEvent.getByToken(bjetsToken_, bTagHandle);
+  const reco::JetTagCollection& bTags = *(bTagHandle.product());
+  std::vector<TLorentzVector> list_bjets;
+  std::cout << "nbjets " << bTags.size() << std::endl;
+
+  if (!bTags.empty()) {
+    for (unsigned bj = 0; bj != bTags.size(); ++bj) {
+      TLorentzVector lv_bjets;
+      lv_bjets.SetPtEtaPhiE(
+          bTags[bj].first->pt(), bTags[bj].first->eta(), bTags[bj].first->phi(), bTags[bj].first->energy());
+      if ((bTags[bj].second > btagFactor_) && (lv_bjets.Pt() > minPt_Jets_))
+        list_bjets.push_back(lv_bjets);
+    }
+    lbj = list_bjets.size();
+  }
+
+  std::cout << "le " << le << "\tlm " << lm << "\tlj " << lj << "\tlbj " << lbj << std::endl;
```

Comment:
```suggestion
```
These look like comment left after your private debug: please remove (and in any case don't use thread unsafe `cout` in centrally run plugins)